logging/upstream_log.go

```diff
@@ -149,17 +152,37 @@ func (u *UpstreamLog) RoundTrip(req *http.Request) (*http.Response, error) {
 		fields["validation"] = validationErrors
 	}
 
+	serverTimingsVal := make(utils.ServerTimings)
+	serverTimingsKey := ""
+
+	// For test cases: "backend" is sometimes not set
+	if u.log != nil {
+		if key, exists := u.log.Data["backend"]; exists {
+			serverTimingsKey = key.(string)
+		}
+	}
+
+	if name, ok := requestFields["name"].(string); ok && name != "default" {
```

Comment:
`configload.defaultNameLabel`, just in case, the default name is changed :-)

Reply:
Done...